test/e2e/suite/conformance/certificates/vault/vault_approle.go

```diff
@@ -47,6 +47,7 @@ var _ = framework.ConformanceDescribe("Certificates", func() {
 		// Vault does not support signing using Ed25519
 		featureset.Ed25519FeatureSet,
 		featureset.IssueCAFeature,
+		featureset.MaxPathLenFeature,
```

Comment:
This is surprising, I though vault _does_ support pathLen

Reply:
> This PR adds support for the CA and SelfSigned issuer. Vault can be extended to support pathLen, but this will require much wider changes to the Vault issuer since we'd need to call a different Vault endpoint to issue CA certificates. That's left for a separate PR.

I added this to the PR description to address this